### ISSUE TITLE
Unable to locate component inside Touchable components in testing due to accessibility being hard coded to true

### ISSUE THREAD

**Issue by krimsonkla** [facebook/react-native#6560]
Appium is unable to locate items inside Touchables due to accessible being hard coded to true as expressed by another user in this stack overflow question. http://stackoverflow.com/questions/35290427/react-native-cant-find-child-elements-when-accessible-is-true

With accessible: true, any element under touchable with a testID does not display in Appium Inspector.

Sample JSX below. testID should appear on 2 items within this rendered JSX. TouchableOpacity (notificationRow) and Text (notificationDescription)
```
<TouchableOpacity
        key={'NotificationItem'}
        onPress={isSelected ? deselect : select}
        style={[style, {
          flex: 1
        }]}
        activeOpacity={0.5}
        testID='notificationRow'
      >
        <View style={[styles.root]}>
          { !iconHidden &&
            <View style={styles.iconContainer}>
              <Image
                style={
                  notification.isRead ?
                  styles.iconRead
                  : styles.icon
                }
                source={notificationIcon}
              />
            </View>
          }
          <View style={styles.details}>
            <Text
              style={styles.text}
              testID={'notificationDescription'}
            >
```
Results in the following in Appium Inspector. (Notice there are no items below the notification row)
<img width="666" alt="screen shot 2016-03-21 at 6 01 51 am" src="https://cloud.githubusercontent.com/assets/354469/13919955/9c0a152c-ef2d-11e5-9913-64a61f98b2c9.png">

With the TouchableOpacity accessible prop set to false, all sub elements are visible in appium inspector. (Notice notification  description is selectable.)
<img width="823" alt="screen shot 2016-03-21 at 6 10 50 am" src="https://cloud.githubusercontent.com/assets/354469/13919960/9e4e7634-ef2d-11e5-9dfc-cc45f86ed1e5.png">

Working on the PR for this now, will comment with PR link when complete.

**Comment by kemurphy** [comment]
Looks like this was intended to be closed in 4f18707?

**Comment by mkonicek** [comment]
Hi there! This issue is being closed because it has been inactive for a while.

But don't worry, it will live on with ProductPains! Check out its new home: https://productpains.com/post/react-native/unable-to-locate-component-inside-touchable-components-in-testing-due-to-accessibility-being-hard-coded-to-true

Product Pains has been very useful in highlighting the top bugs and feature requests:
https://productpains.com/product/react-native?tab=top

Also, if this issue is a bug, **please consider sending a pull request with a fix**.
